osotiihme_assignment2.ipynb

```diff
@@ -0,0 +1,1992 @@
+{
```

Comment:
<p>This is cool doing the plot in 2 ways! There are aways so many ways to do things that it's neat to see other approaches, great work! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research/pull/1705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>